### ISSUE TITLE
Any timeframe for the next stable version?

### ISSUE THREAD

**Issue by davidkalosi** [beberlei/litecqrs-php#47]
@beberlei any idea when the master branch will get into consolidated state ?

**Comment by tyx** [comment]
Hi !

I'm very interessed by this question too. We started to use it (dev-master) for one project and hope we could continue with support !

**Comment by davidkalosi** [comment]
Another reason why I am asking is I have made bridge that will replay the events into a rabbitmq queue. 
We need to push domain events outside PHP/symfony so that other systems can act on them. 
It's currently implemented to the 1.1 branch by an ReplayingEventBus implementation of the ContainerEventBus that always adds a RabbitMqHandler to the handler list. 

I am using a decorated domain message that adds fields specific to the rabbitmq like the topic name etc ... 
The handler acts using the __call method on the decorated messages and pushes them into the rabbit. 

The current implementation is sort of a POC and since the master branch is almost completely refactored I would prefer to continue work on the new release. 

If this feature would sound interesting I would be happy to contribute it to the project.

**Comment by mbadolato** [comment]
@davidkalosi That definitely sounds like something that would be useful. Perhaps it could be contributed as a plugin to LiteCQRS?

**Comment by davidkalosi** [comment]
@mbadolato that was my initial taught. just need to know if the dev-master branch is in a usable state as I would prefer to work on this rather then the 1.1 which  will be obsolete soon :)

**Comment by DracoBlue** [comment]
Any updates on this?

**Comment by davidkalosi** [comment]
@DracoBlue I have started my own project based on the Axon Java framework aiming to create a full blown CQRS library. Here is the link to the repo https://github.com/davidkalosi/GovernorFramework.

It's still in heavy WIP but I already got quite a lot of functionality covered and it is in active development because it will be the backbone of a commercial application my team is working on at the moment. I except to reach a stable and publishable version within 5-6 months.
